blocks/hero/hero.css

```diff
@@ -30,13 +30,18 @@ main .hero .hero-content > div {
 
 main .hero-container .hero-wrapper {
   padding: 0;
-  border-bottom: 32px solid #e4f2ff;
 }
 
 main .hero {
   position: relative;
   padding: 0;
   min-height: 300px;
+  border-bottom: 32px solid #e4f2ff;
```

Comment:
For colors, might worth defining variables either in block css or globally if other block are using that.